.github/workflows/discourse-plugin.yml

```
@@ -55,7 +55,7 @@ jobs:
         run: |
           yarn prettier -v
           if [ 0 -lt $(find assets admin/assets -type f \( -name "*.scss" -or -name "*.js" -or -name "*.es6" -or -name "*.hbs" \) 2> /dev/null | wc -l) ]; then
-            yarn prettier --list-different "assets/**/*.{scss,js,es6,hbs}"
+            yarn prettier --list-different "**/assets/**/*.{scss,js,es6,hbs}"
```

This might be more performant (i.e. should prevent prettier from rummaging through all of node_modules)

```suggestion
            yarn prettier --list-different "{.,admin}/assets/**/*.{scss,js,es6,hbs}"
```

---

Let's split it into two if+prettier blocks: one for `assets` and one for `admin/assets`